CONTRIBUTING.md

```diff
@@ -124,7 +124,7 @@ provide as much detail and context as possible.
 
 ## Pull requests
 
-Good pull requests—patches, improvements, new features—are a fantastic
+Good pull requests, patches, improvements, and new features are all fantastic
```

Comment:
"Good pull requests" isn't a separate item, it's a category under which the others are examples. The dashes act like parentheses in this case.
See http://www.grammarly.com/handbook/punctuation/dash/2/dash-parenthetical-information/